cmd/otelopscol/main.go

```diff
@@ -44,6 +47,12 @@ func main() {
 	params := service.CollectorSettings{
 		Factories: factories,
 		BuildInfo: info,
+		LoggingOptions: []zap.Option{
+			levelchanger.NewLevelChangerOption(
+				zapcore.ErrorLevel,
+				zapcore.DebugLevel,
+				levelchanger.FilePathLevelChangeCondition("scrapercontroller.go")),
```

Comment:
Can you please point to the following line and say something like: this is the only error that gets mutated
https://github.com/open-telemetry/opentelemetry-collector/blob/831373ae6c6959f6c9258ac585a2ec0ab19a074f/receiver/scraperhelper/scrapercontroller.go#L198